packages/calypso-e2e/src/lib/components/navbar-component.ts

```diff
@@ -27,4 +31,30 @@ export class NavbarComponent extends BaseContainer {
 	async clickMySites(): Promise< void > {
 		await this.page.click( selectors.mySiteButton );
 	}
+
+	/**
+	 * Opens the notification panel.
+	 *
+	 * Optionally, set the `keyboard` parameter to trigger this action using keyboard shortcut.
+	 *
+	 * @param {{[key: string]: string}} [param0] Object assembled by the caller.
+	 * @param {string} param0.keyboard Set to true to use keyboard shortcut to open the notifications panel. Defaults to false.
+	 * @returns {Promise<void>} No return value.
+	 */
+	async openNotificationsPanel( {
+		keyboard = false,
```

Comment:
Currently, the only place this is used is in the `notifications-trash` spec where the panel is opened using keyboard shortcut.

Reply:
nit
```suggestion
		useKeyboard = false,
```

Reply:
Addressed, thanks!